Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+MAINTAINER Devin Ekins <devops@keen.io>
+
+# Setup the dependencies
+RUN apt-get update -y
+RUN apt-get install -y apt-utils
```

Comment:
When I did this, the latter tools spewed warnings because `apt-utils` is used to properly configure the later tools.  When I split it into multiple commands, that warning goes away.